README.md

```diff
@@ -13,24 +13,24 @@
 ├── .gitignore
 |
 ├── data
-│   ├── demofile.json
-│   └── testfile.json
+│   ├── demofile.json
```

Comment:
I do not believe that demofile.json is still in the repository, so it probably does not need to be in the readme.

Reply:
Thanks for pointing that out, although it looks like some changes were made to the README file, there were no direct changes from the one currently in master. We are trying to keep it this way because there are other teams working on changing the README file and we don't want to cause any merge conflicts for them. I completely agree with you, however.